apps/web/src/pages/ApplicantDashboardPage/components/FunctionalCommunityListItem.tsx

```diff
@@ -0,0 +1,196 @@
+}
+
+const FunctionalCommunityListItem = ({
+  headingAs,
+  functionalCommunityListItemFragment,
+}: FunctionalCommunityListItemProps) => {
+  const intl = useIntl();
+
+  const sharedIconStyling = {
+    "data-h2-height": "base(x1)",
+    "data-h2-width": "base(x1)",
+    "data-h2-display": "base(inline-block)",
+    "data-h2-vertical-align": "base(bottom)",
+    "data-h2-margin-right": "base(x.25)",
+    "data-h2-padding-top": "base(x.125)",
+  };
+
+  const interestedWork = (
+    <span>
+      <BriefcaseIcon
+        data-h2-color="base(success)"
+        {...sharedIconStyling}
+      ></BriefcaseIcon>
+      {intl.formatMessage({
+        defaultMessage: "Interested in work",
+        id: "1VKNrs",
+        description: "Phrase marking interest in community work opportunities",
+      })}
+    </span>
+  );
+  const notInterestedWork = (
+    <span>
+      <XCircleIcon
+        data-h2-color="base(gray.lighter)"
+        {...sharedIconStyling}
+      ></XCircleIcon>
+      {intl.formatMessage({
+        defaultMessage: "Not interested in work",
+        id: "VDVRPt",
+        description:
+          "Phrase marking lack of interest in community work opportunities",
+      })}
+    </span>
+  );
+  const missingWork = (
+    <span>
+      <ExclamationTriangleIcon
+        data-h2-color="base(error)"
+        {...sharedIconStyling}
+      ></ExclamationTriangleIcon>
+      {intl.formatMessage({
+        defaultMessage: "<red>Missing work info</red>",
+        id: "ZrLeP8",
+        description:
+          "Phrase marking incomplete community work opportunities interest",
+      })}
+    </span>
+  );
+
+  const generateMetaDataJobInterest = (
+    jobInterest: boolean | null | undefined,
+  ): JSX.Element => {
+    if (jobInterest === null || jobInterest === undefined) {
+      return missingWork;
+    }
+    return jobInterest ? interestedWork : notInterestedWork;
+  };
+
+  const interestedTraining = (
+    <span>
+      <PresentationChartLineIcon
+        data-h2-color="base(success)"
+        {...sharedIconStyling}
+      ></PresentationChartLineIcon>
+      {intl.formatMessage({
+        defaultMessage: "Interested in training",
+        id: "ERsZAD",
+        description:
+          "Phrase marking interest in community training opportunities",
+      })}
+    </span>
+  );
+  const notInterestedTraining = (
+    <span>
+      <XCircleIcon
+        data-h2-color="base(gray.lighter)"
+        {...sharedIconStyling}
+      ></XCircleIcon>
+      {intl.formatMessage({
+        defaultMessage: "Not interested in training",
+        id: "8wU0cq",
+        description:
+          "Phrase marking lack of interest in community training opportunities",
+      })}
+    </span>
+  );
+  const missingTraining = (
+    <span>
+      <ExclamationTriangleIcon
+        data-h2-color="base(error)"
+        {...sharedIconStyling}
+      ></ExclamationTriangleIcon>
+      {intl.formatMessage({
+        defaultMessage: "<red>Missing training info</red>",
```

Comment:
Consider removing the <red> formatting from the string and doing it on the `<p>` element instead so the string can be reused.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/commit/9368ed0dab4b494a18a55b8fefc2ac88e899f0b8